.changeset/fifty-zebras-check.md

```diff
@@ -0,0 +1,5 @@
+---
+'@propeldata/ui-kit': minor
+---
+
+feat(metric): add ability to query metrics on-the-fly
```

Comment:
Can we remove this prefix? The rest of the changesets don't have it, just to keep consistency
```suggestion
Added ability to query metrics on-the-fly
```